Update to remove Google Groups (#7664)
vitessio/vitess: SECURITY.md

**Disclosure of Forthcoming Fix to Users** (Completed within 1-7 days of Disclosure)

- The Fix Lead will update Vitess Slack informing users that a security vulnerability has been disclosed and that a fix will be made available at YYYY-MM-DD HH:MM UTC in the future via this list. This time is the Release Date.
- The Fix Lead will include any mitigating steps users can take until a fix is available.

The communication to users should be actionable. They should know when to block time to apply
patches, understand exact mitigation steps, etc.

**Disclosure of Fixed Vulnerability**

> - The Fix Lead will update Vitess Slack informing users that there are new releases available to address an identified vulnerability.  - As much as possible this email should be actionable and include links to CVEs, and how to apply the fix to user's environments; this can include links to external distributor documentation.

Reviewer: ```suggestion
- The Fix Lead will post a notice on Vitess Slack informing users that there are new releases available to address an identified vulnerability.  - As much as possible this notice should be actionable and include links to CVEs, and how to apply the fix to user's environments; this can include links to external distributor documentation.
```
